src/components/ToggleButtons/ToggleButtons.jsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable react/prop-types */
+import { useState } from 'react';
+import Button from '../Button/Button';
+import { FlexContainer } from '../FlexContainer/FlexContainer';
+import './ToggleButtons.css';
+
+const ToggleButtons = ({ filterByCategory }) => {
+    const [selectedButton, setSelectedButton] = useState('dessert');
+    const buttonData = [
```

Comment:
You need to get categories as unique values of categories from meal list, not to hard code it